apisix/cli/ngx_tpl.lua

```diff
@@ -26,6 +26,12 @@ worker_processes {* worker_processes *};
 worker_cpu_affinity auto;
 {% end %}
 
+# main start configuration snippet starts
```

Comment:
I think we can directly move `main_configuration_snippet` here instead of creating new one?